### ISSUE TITLE
[Bug][a11y][Grid View]Tab index of total open tabs not read 

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#16477]
## Prerequisites
Talkback should be enabled
Have a couple couple of open tabs

## Steps to reproduce
1. Open the Tabs Tray
2. Tap on the first listed tab
3. Tap on the second listed tab

### Expected behavior
Step 2: 1 of x (number of open tabs)
Step 3: 2 of x (number of open tabs)

### Actual behavior
The tab index of total open tabs is not read

### Device information

* Android device: Google Pixel 3a (Android 11)
* Fenix version: Nightly Build 201110

**Comment by mcarare** [comment]
As a note, also to be taken into consideration for QA when checking this: The expected behavior for a grid view item is that Talkback announces the column and row the item occupies in the grid. 
For example: " Row 1, Column 1" accompanied by "In grid 3 rows, 2 columns" only when selecting a tab after the previously selected element is outside the grid. Also, the column or row number is announced only when the row of the column is changed. For example, after " Row 1, Column 1" if an item in the same row is selected only "Column 2" will be announced.

**Comment by sflorean** [comment]
Verified as fixed on Nightly 11/17 with Samsung Galaxy Note10 (Android 10).
